pkg/platform/platform.go

```diff
@@ -513,6 +515,63 @@ func (platform *Platform) WaitForNamespace(ns string, timeout time.Duration) {
 	k8s.WaitForNamespace(client, ns, timeout)
 }
 
+func GenerateBaseYaml(baseYamlContent string) {
+	kustWorkingDir := "overlays/baseDir/"
+	os.RemoveAll(kustWorkingDir)
+	err := os.MkdirAll(kustWorkingDir, 0755)
+   	if err != nil {
+    	log.Fatal(err)
+    }
+    err = ioutil.WriteFile(kustWorkingDir+"base.yaml", baseYamlContent, 0644)
+	if err != nil {
+		log.Fatal(err)
+	}
+}
+
+func GetFinalPatchYaml(patchFilePath string) string {
+	kustWorkingDir := "overlays/patchDir/"
+	os.RemoveAll(kustWorkingDir)
+	err := os.MkdirAll(kustWorkingDir, 0755)
+   	if err != nil {
+    	log.Fatal(err)
```

Comment:
Could you run your IDE, text editor with go tools and follow the best practices? the function has spaces that are not accepted in golang. Please use `gofmt` and run automatically.

Reply:
sure, done formatting.